packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py

```diff
@@ -234,6 +239,18 @@ def __call__(self, message: str) -> bool:
                     round(progress, 1) == 1.0
                     for progress in self._current_progress.values()
                 )
+
+        _current_timestamp = datetime.now(UTC)
+        if _current_timestamp - self._last_poll_timestamp > timedelta(seconds=5):
+            url = f"https://{self.node_id}.services.{self.get_partial_product_url()}"
+            response = httpx.get(url, timeout=10)
+            self.logger.info(
+                f"Querying the service endpoint from the E2E test. Response: {response}"
+            )
+            if response.status_code == 200:
+                return True
```

Comment:
could you maybe add a warning here in case the progress was not complete? thanks

Reply:
added